data/Makefile.am

```diff
@@ -124,6 +124,7 @@ EXTRA_DIST = proj.ini GL27 nad.lst nad27 nad83 \
 		tests/simple_model_wrap_west.tif \
 		tests/simple_model_projected.json \
 		tests/tinshift_crs_implicit.json \
+		tests/tinshift_simplified_etrs89_ngo48.json \
```

Comment:
I don't really think we need this file. The purpose of the existing tests/tinshift_ was to test the tinshift mechanism itself. Adding tinshift_simplified_etrs89_ngo48.json doesn't bring much.